docs/user/implement_algo.rst

```diff
@@ -348,9 +358,107 @@ For completeness, the full experiment file is repeated below:
 
     debug_my_algorithm()
 
+TensorFlow
+----------
+
+The TensorFlow version is similar to PyTorch except for building the
+computation graph and the :code:`train_once()` function.
+
+.. code-block:: python
+
+    import tensorflow as tf
+    import numpy as np
+
+    from garage import log_performance, TrajectoryBatch
+    from garage.sampler import RaySampler
+    from garage.misc import tensor_utils
+
+    class MyAlgorithm():
+
+        sampler_cls = RaySampler
+
+        def __init__(self, env_spec, policy):
+            self.env_spec = env_spec
+            self.policy = policy
+            self.max_path_length = 200
+            self._lr = 1e-3
+            self._discount = 0.99
+
+            obs_space = self.policy.observation_space
+            action_space = self.policy.action_space
+            with tf.name_scope('inputs'):
+                self._obs = obs_space.to_tf_placeholder(name='obs', batch_dims=2)
```

Comment:
Would this file be simpler if we only used one batch dimension and we concatenated (instead of padding and stacking) the trajectories? I don't have a very strong opinion, but we should try and keep this example relatively simple.